### ISSUE TITLE
Message bar overflows from window when plugin is enabled

### ISSUE THREAD

**Issue by hibifier** [programmer2514/BetterDiscord-CollapsibleUI#143]
**Describe the bug**
The message bar escapes the Discord window when you type long messages.

**To Reproduce**
Type a message long enough that the message bar has to be resized. This should work without fail regardless of theme and setting.

**Expected behavior**
The message bar should stay within the bounds of the Discord window [Screenshot 2].

**Screenshots**
![image](https://github.com/user-attachments/assets/0f2c54b5-5387-4937-839e-9200db500908)
_Screenshot 1. Bugged behavior. Notice how the message bar goes past the slow mode indicator and escapes the Discord window._

![image](https://github.com/user-attachments/assets/e9693fbe-24b0-481a-8a5e-31ad03f95d04)
_Screenshot 2. Expected behavior._

**Desktop and Discord information**
 - OS: Windows 11
 - Discord Version: Stable
 - Enabled Theme(s): AMOLED-Cord

**Comment by glae-millen** [comment]
Yes. At what length of text it actually occurs seems to be erratic though.

![image](https://github.com/user-attachments/assets/1172a126-d20b-4775-b68a-98d7ba948c43)
![image](https://github.com/user-attachments/assets/048f93db-68b4-478f-9755-a04a41ac31d8)
